provider/openstack/provider.go

```diff
@@ -2080,7 +2080,7 @@ func rulesToRuleInfo(groupId string, rules firewall.IngressRules) []neutron.Rule
 		}
 		sourceCIDRs := r.SourceCIDRs.Values()
 		if len(sourceCIDRs) == 0 {
-			sourceCIDRs = append(sourceCIDRs, firewall.AllNetworksIPV4CIDR)
+			sourceCIDRs = append(sourceCIDRs, firewall.AllNetworksIPV4CIDR, firewall.AllNetworksIPV6CIDR)
```

Comment:
assume this will be ok on clouds that don't support v6?

Reply:
yeah Openstack doesn't care.